package.json

```diff
@@ -48,6 +49,7 @@
   "dependencies": {
     "@contentful/app-scripts": "file:packages/contentful--app-scripts",
     "@contentful/create-contentful-app": "file:packages/contentful--create-contentful-app",
-    "create-contentful-app": "file:packages/create-contentful-app"
+    "create-contentful-app": "file:packages/create-contentful-app",
+    "@contenful/react-apps-toolkit": "file:packages/react-apps-toolkit"
```

Comment:
We are dropping these dependencies in another PR. Maybe those are causing your issues